vagga.yaml

```diff
@@ -24,3 +46,26 @@ commands:
     description: Shortcut to manage.py
     container: django
     run: [python3, manage.py]
+  run-postgres: !Supervise
+    description: Start the django development server using Postgres database
+    children:
+      app: !Command
+        container: django
+        environ:
+          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/django
+        run: |
+            touch /work/.dbcreation # Create lock file
```

Comment:
I would follow python's conventions here as well and use 2 whitespaces before `#`